Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -1715,6 +1715,22 @@ void dwinPrintFinished() {
 
 // Print was aborted
 void dwinPrintAborted() {
+  #if ENABLED(NOZZLE_PARK_FEATURE)
+    const xyz_pos_t park_point = NOZZLE_PARK_POINT;
+  #endif
+  if (all_axes_homed()) {
+    const int16_t zpos = current_position.z + TERN(NOZZLE_PARK_FEATURE,
+    NOZZLE_PARK_Z_RAISE_MIN, Z_POST_CLEARANCE);
+    _MIN(zpos, Z_MAX_POS);
+    MString<25> cmd;
+    cmd.setf(cmd, F("G0Z%i\nG0F2000Y%i"), zpos, TERN(NOZZLE_PARK_FEATURE, park_pos.y, 200));
+    queue.inject(&cmd);
+  }
+  #ifdef SD_FINISHED_RELEASECOMMAND
```

Comment:
`EVENT_GCODE_SD_ABORT` should be used here instead.

Reply:
I agree

Reply:
In fact, this should be removed altogether. The `SD_FINISHED_RELEASECOMMAND` G-code is properly invoked from within `M1001` as part of SD print completion in the proper circumstances, allowing for a wider variety of G-codes to be used safely as part of `SD_FINISHED_RELEASECOMMAND`.